lib/vm.py

```diff
@@ -338,6 +331,18 @@ def destroy_vdi(self, vdi_uuid: str) -> None:
                 super().destroy_vdi(vdi_uuid)
                 break
 
+    def create_vdis_list(self) -> None:
```

Comment:
`_create_vdis_list` would make it more obvious this is an internal method

Reply:
It's still not for users of the object, just like `_disk_list`.  One leading underscore is the conventional equivalent to C++' `protected`, there two of them are the `private` equivalent.